examples/volumes/flexvolume/dummy-attachable

```diff
@@ -0,0 +1,104 @@
+#!/bin/sh
+
+# Copyright 2017 The Kubernetes Authors.
```

Comment:
okay. (I will move them to test/e2e/manifests, not examples/, since atm I'm relying on this script to compile the contents of examples/ https://github.com/kubernetes/kubernetes/blob/master/hack/generate-bindata.sh#L48)